tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -111,16 +111,16 @@ export class RunsDataTable {
   }
 
   allRowsSelected() {
-    return this.data.every((row) => row['selected']);
+    return this.data?.every((row) => row['selected']);
```

Comment:
These should never be `undefined` or `null` as long as the store exists, however, we have a funky test in `tensorboard/webapp/metrics/metrics_integration_test.ts` which checks the state of the app if the store is not defined.